frontend/app/services/personal-information-service.server.ts

```diff
@@ -212,6 +214,9 @@ function createPersonalInformationService() {
   async function getPersonalInformation(personalSinId: string) {
     const curentPersonalInformation = createClientInfo(personalSinId);
     const url = `${INTEROP_POWERPLATFORM_API_BASE_URI}/dental-care/applicant-information/dts/v1/applicant/`;
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+    auditService.audit('personal-information.get', { userId: 'anonymous' });
```

Comment:
We should log either the `clientId` here, or some other identifier that we can use to find the audit event in the future.

Reply:
Fixed: fe83f7777d7d1765885535ec99e2b31d758b8fb6

Reply:
@nicholas-ly again.. do you think this should be the RAOIDC `sub` claim?

Reply:
Fixed as per previous discussions: b54fdc1071cc2f7076d56ba903332d1f0b525eed